lib/module-three/src/pipes/square.pipe.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { async, TestBed } from '@angular/core/testing';
+import { SquarePipe } from './square.pipe';
+
+describe('Pipe: SquarePipe', () => {
+    let pipe: SquarePipe;
```

Comment:
4 space indentation should be 2 space indentation.
Please also add this rule to tslint.json